src/script/interpreter.cpp

```diff
@@ -459,6 +471,22 @@ bool CheckDataSignatureEncoding(const valtype &vchSig, uint32_t flags, ScriptErr
     return CheckSignatureEncodingSigHashChoice(vchSig, flags, serror, false);
 }
 
+bool CheckTransactionECDSASignatureEncoding(const valtype &vchSig, uint32_t flags, ScriptError *serror)
+{
+    // Insist that this sig is Schnorr
+    if (vchSig.size() == 65)
+        return set_error(serror, SCRIPT_ERR_SIG_BADLENGTH);
+    return CheckSignatureEncodingSigHashChoice(vchSig, flags, serror, true);
+}
+
+bool CheckTransactionSchnorrSignatureEncoding(const valtype &vchSig, uint32_t flags, ScriptError *serror)
+{
+    // Insist that this sig is Schnorr
```

Comment:
outdated comment?

Reply:
right, the comment is trying to say that this is the code that errors out if not schnorr because we know schnorr must be 65 bytes.  Maybe the problem with the comment is that it describes something that's obvious from the code...